.php_cs

```diff
@@ -1,12 +1,17 @@
 <?php
 
-$finder = Symfony\CS\Finder\DefaultFinder::create()
-    ->exclude('pattern-library')
+$finder = PhpCsFixer\Finder::create()
     ->name('update')
+    ->exclude('var')
```

Comment:
Looks like we should ignore `cache`, `src/eLife/Medium/Model/Map` and `src/eLife/Medium/Model/Base` too.

Reply:
I don't have these folders locally, and `cache` is in `var/cache/` (if it's that one). Something needs to run for them to be created?

Reply:
`cache` exists on `prod` (and `.gitignore`), but looks like it was moved to `var/cache`. The others are created by Propel.

Reply:
It looks like they are in `var/cache/propel/src/eLife/Medium/Model` now?

Reply:
💡